libcontainer/cgroups/fs2/create.go

```diff
@@ -100,8 +100,14 @@ func CreateCgroupPath(path string, c *configs.Cgroup) (Err error) {
 		if i < len(elements)-1 {
 			file := filepath.Join(current, "cgroup.subtree_control")
 			if err := ioutil.WriteFile(file, []byte(allCtrs), 0755); err != nil {
-				// XXX: we can enable _some_ controllers doing it one-by one
-				// instead of erroring out -- does it makes sense to do so?
+				if rootless {
+					// try write one by one
+					for _, ctr := range ctrs {
+						_ = ioutil.WriteFile(file, []byte(ctr), 0755)
+					}
+					// ignore unwritable files
```

Comment:
1. We are writing to the same file here, so this comment is misleading.

2. Enabling controllers one by one makes sense if some controllers can't be enabled by the kernel (for example, pids and memory controller can't be enabled if we already have processes in this cgroup, while others can still be enabled).

I don't think that in case of rootless you can enable some controllers but not some others. Unless I'm wrong, it does not make sense to try to enable them individually here.

Reply:
> Enabling controllers one by one makes sense if some controllers can't be enabled by the kernel 

And if constrained by the parent subtree_control